lib/cli/run.js

```diff
@@ -176,6 +176,10 @@ exports.builder = yargs =>
         group: GROUPS.OUTPUT,
         hidden: true
       },
+      'node-option': {
+        description: 'Node or V8 option (no leading "--")',
```

Comment:
How about `no leading "--" or "--v8-"`

Reply:
I haven't added the `--v8-` part, see comment above.